liquidhaskell.cabal

```diff
@@ -271,4 +274,6 @@ test-suite test
                      tasty >= 0.10,
                      tasty-hunit >= 0.9,
                      tasty-rerun >= 1.1,
-                     transformers
+                     transformers,
+                     Cabal >= 1.18
```

Comment:
We should only need `Cabal` as a dependency in the `library` section since we don't import any of its modules in `Liquid.hs` or `test.hs`.